sys/dev/athn/if_athn_usb.c

```diff
@@ -578,13 +578,13 @@ athn_usb_attachhook(device_t self)
 	}
 
 	// TODO MichalP: this can be used as a starting point for echo command or firmware command
-//	mtx_lock(&usc->sc_sc.sc_mtx);
+//	ATHN_LOCK(usc);
```

Comment:
Shouldn't it be `ATHN_LOCK(&usc->sc_sc);`.?

Reply:
Indeed, you have the right. Change is on the way :)